portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path

```diff
@@ -95,6 +95,11 @@
 	<td></td>
 </tr>
 <!--TOOLBAR-->
+<tr>
```

Comment:
Hi @piresdm, since Diogo and Renato would like to keep paths and macros within Form files, could you please move this? :D Thank you!

Reply:
Thank you! :D